ui/admin/app/styles/app.scss

```diff
@@ -398,51 +398,53 @@
 .worker-create-section {
   border: sizing.rems(xxxxs) solid var(--ui-gray-subtler-3);
   border-radius: sizing.rems(xxs);
-  padding: sizing.rems(l);
   margin-bottom: sizing.rems(xl);
+  padding: sizing.rems(l);
 
   > div {
     margin-bottom: sizing.rems(l);
   }
 
   > code {
-    color: var(--failure);
     background-color: var(--ui-gray-subtler-5);
     border: sizing.rems(xxxxs) solid var(--ui-gray-subtler-2);
+    color: var(--failure);
   }
 
   .worker-auth-token {
-    display: flex;
-    gap: sizing.rems(m);
     align-items: flex-end;
+    display: flex;
+    gap: sizing.rems(m); // sass-lint:disable-line no-misspelled-properties
```

Comment:
@ZedLi linter says `gap` is misspelled, suggests to use `row-gap`. I have no context with the implementation of these styles, so I wanted to check with you before changing the `gap` to `row-gap` or to discuss possible solutions.

Reply:
I think that `gap` allows you to set both spacing for the `column`  and `row` without having to specifically say `column-gap` or `row-gap`

Reply:
Yes that's right, it's just shorthand for both `row-gap` and `column-gap`, I wonder if our linter needs to just be updated to pick the syntax?

Reply:
All good! 😄 In this case I just add the one liner to deactivate such rule. Not a fan of this practice but the code makes sense, so better to wait for the linter to be updated (they are already working on it).
Thanks for the info guys 👍